### PR TITLE
Updated code contribution guideline in README to reflect the neccessi…

### PR DESCRIPTION
**Pull request by buildscientist**
…ty of including Rspec tests with any changes made.

Resolves issue #3.

### CODE REVIEW

**Comment by buildscientist** [comment]
Also updated the .gitignore file with one that was generated from [GitIgnore.io](https://www.gitignore.io/api/rubymine,ruby,vim,osx,windows,linux,sublimetext,textmate)

**Comment by sjaveed** [comment]
That .gitignore is quite a handful :-)  Probably covers more cases than I need but seems reasonable.

File: .gitignore [hunk 1]
```diff
@@ -1,18 +1,179 @@
...
+# Crashlytics plugin (for Android Studio and IntelliJ)
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+
+
+### Ruby ###
 *.gem
 *.rbc
-.bundle
```
Comment (sjaveed) [inline .gitignore:3]: Some of these are rails-specific ignores.  Could you update your PR so it's a super-set of everything in the original file?

**Comment by sjaveed** [comment]
:+1: Merging.

**Comment by sjaveed** [comment]
Thanks @buildscientist!

**Pull request merged**